jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_sensors/baxter_spherical_kodak.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
+  <arg name="spherical_camera_ns" default="kodak_head" />
+
+  <node name="kodak_transform_publisher" pkg="tf" type="static_transform_publisher"
```

Comment:
Please add `tf` to package.xml (although it is already required by `dynamic_tf_publisher`)